apiserver/billing/odoostorage/update.go

```diff
@@ -38,5 +40,13 @@ func (s *billingEntityStorage) Update(ctx context.Context, name string, objInfo
 		}
 	}
 
+	if !reflect.DeepEqual(newBE.Spec, oldBE.Spec) {
+		apimeta.SetStatusCondition(&newBE.Status.Conditions, metav1.Condition{
```

Comment:
I'd have preferred something a bit more generic like a `lastUpdated` (maybe also using Odoo `__last_update`) field in the status but this is fine for now.

Reply:
Problem here is that changing the status updates the record, so there's loop potential.